.github/workflows/tests.yaml

```diff
@@ -15,19 +15,16 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: '16'  # Use Node.js 16.x 
+          node-version: '18'  # Use Node.js 18.x 
 
       - name: Install pnpm
-        run: npm install -g pnpm
-      
-      - name: Changing Directory
-        run: cd nepalingo-web
+        run: npm -C nepalingo-web install -g pnpm
```

Comment:
Let's use this pnpm action: https://pnpm.io/continuous-integration#github-actions